app/src/main/java/org/mifos/mobile/models/payload/LoginPayload.kt

```diff
@@ -0,0 +1,16 @@
+package org.mifos.mobile.models.payload
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Created by Ashwin on 23rd June 2020
+ */
+
+data class LoginPayload(
+
+        @SerializedName("username")
```

Comment:
Cannot we add Serialization at class-level and use of SerializedName is not needed since the variable name and SerializedName are the same. Please clarify.

Reply:
@miPlodder Yes, `@SerializedName` does not seem quite needed. I took reference from `RegisterPayload.kt` so that this matches with the other Payload classes where the author seemed to have used Serialized names with the same name as that of the variable. 
But removing that doesn't cause a problem. If it is fine to not follow the same style as that of the other payload classes, shall I proceed to remove it?

Reply:
We can remove it when we are migrating those functionality

Reply:
Sure, it is removed for this class now. I will keep this in mind while converting the rest.